openmetadata-ui/src/main/resources/ui/src/rest/feedsAPI.ts

```diff
@@ -189,6 +191,32 @@ export const getActiveAnnouncement = async (entityLink?: string) => {
 
   return response.data;
 };
+export const getAnnouncements = async (
+  activeAnnouncement?: boolean,
+  entityLink?: string
+) => {
+  const params: {
+    type: ThreadType;
+    activeAnnouncement?: boolean;
+    entityLink?: string;
+  } = {
+    type: ThreadType.Announcement,
+    activeAnnouncement: activeAnnouncement,
+  };
+
+  if (entityLink) {
+    params.entityLink = entityLink;
+  }
```

Comment:
I don' think if we needed this condition, we can use it directly in the object. can you have a look ?

Reply:
yes, corrected this, using directly in the object